src/signals/incident-management/components/FilterForm/index.js

```diff
@@ -353,352 +374,361 @@ const FilterForm = ({
+                checked={state.filter.refresh}
+                id="filter_refresh"
+                name="refresh"
+                onClick={onRefreshChange}
+              />
+              <AscLabel
+                htmlFor="filter_refresh"
+                label={
+                  <Fragment>
+                    <RefreshIcon width={16} height={18} /> Automatisch verversen
+                  </Fragment>
+                }
+              />
+            </div>
+          </Fieldset>
+
+          <Fieldset>
+            <FilterGroup>
+              <Label htmlFor="filter_notes" isGroupHeader>
+                Zoek in notitie
+              </Label>
+              <Input
+                data-testid="filterNotes"
+                name="note_keyword"
+                id="filter_notes"
+                onBlur={onNoteBlur}
+                onChange={onNoteChange}
+                value={controlledTextInput.note}
+                type="text"
+              />
+            </FilterGroup>
+          </Fieldset>
+
+          <Fieldset>
+            <legend>Filter parameters</legend>
+
+            <CheckboxGroup
+              defaultValue={state.options.status}
+              label="Status"
+              name="status"
+              onChange={onGroupChange}
+              onToggle={onGroupToggle}
+              onSubmit={onSubmitForm}
+              options={dataLists.status}
             />
-          </FilterGroup>
-        </Fieldset>
-
-        <Fieldset>
-          <legend>Filter parameters</legend>
-
-          <CheckboxGroup
-            defaultValue={state.options.status}
-            label="Status"
-            name="status"
-            onChange={onGroupChange}
-            onToggle={onGroupToggle}
-            onSubmit={onSubmitForm}
-            options={dataLists.status}
-          />
-
-          {configuration.featureFlags.fetchDistrictsFromBackend &&
-            districts && (
+
+            {configuration.featureFlags.fetchDistrictsFromBackend &&
+              districts && (
+                <CheckboxGroup
+                  defaultValue={state.options.area}
+                  label={configuration.language.district}
+                  name="area"
+                  onChange={onGroupChange}
+                  onToggle={onGroupToggle}
+                  onSubmit={onSubmitForm}
+                  options={districts}
+                />
+              )}
+
+            {!configuration.featureFlags.fetchDistrictsFromBackend && (
               <CheckboxGroup
-                defaultValue={state.options.area}
-                label={configuration.language.district}
-                name="area"
+                defaultValue={state.options.stadsdeel}
+                label="Stadsdeel"
+                name="stadsdeel"
                 onChange={onGroupChange}
                 onToggle={onGroupToggle}
                 onSubmit={onSubmitForm}
-                options={districts}
+                options={dataLists.stadsdeel}
```

Comment:
same here

Reply:
Github does weird comparing here. Everything still in place.